src/GitVersionCore/GitVersionCalculator.cs

```diff
@@ -75,13 +76,13 @@ private string ResolveCurrentBranch(IBuildServer buildServer, string targetBranc
             return currentBranch;
         }
 
-        private VersionVariables GetCachedGitVersionInfo(string targetBranch, string commitId, Config overrideConfig, bool noCache)
+        private VersionVariables GetCachedGitVersionInfo(string targetBranch, string commitId, Config overrideConfig, bool noCache, bool isCiBuild)
```

Comment:
Shouldn't we stick to either `isLocalBuild` or `isCiBuild` to avoid confusion?

```suggestion
        private VersionVariables GetCachedGitVersionInfo(string targetBranch, string commitId, Config overrideConfig, bool noCache, bool isLocalBuild)
```

Reply:
Was indecisive on wording I think `isLocalBuild` is best. I must have missed this one.